src/lib/components/table/terra-table-data-source.ts

```diff
@@ -53,25 +55,25 @@ import { TerraPagerInterface } from '../pager/data/terra-pager.interface';
  * }
  * ```
  */
-export abstract class TerraTableDataSource<T> extends DataSource<T> {
+export abstract class TerraTableDataSource<D, F = any> extends DataSource<D> {
```

Comment:
`F = any` oder doch lieber `F = unknown` ?
```suggestion
export abstract class TerraTableDataSource<D, F = unknown> extends DataSource<D> {
```

Reply:
Das funktioniert leider nicht. Gibt nen TypeError

Reply:
Deswegen muss es leider F = any sein